editor/modes/visual-editor/block-drop-zone.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+import { reduce, get, find } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { DropZone } from '@wordpress/components';
+import { getBlockTypes } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { insertBlocks } from '../../actions';
+
+function BlockDropZone( { index, ...props } ) {
+	const dropFiles = ( files, position ) => {
+		const transformation = reduce( getBlockTypes(), ( ret, blockType ) => {
```

Comment:
This won't work because I'm returning the transformation and not the blockType, the return value of `find` callback is considered a boolean right?